src/main/java/com/fakng/fakngagrgtr/parser/stripe/StripeParser.java

```diff
@@ -0,0 +1,132 @@
+    }
+
+    @Override
+    protected String getCompanyName() {
+        return "Stripe";
+    }
+    @Override
+    public List<Vacancy> getAllVacancies() throws IOException {
+        HtmlPage page = downloadPage(String.format(url, 0));
+        int pageCount = getPageCount(page);
+        List<Vacancy> vacancyList = new ArrayList<>();
+
+        for (int i = 0; i <= pageCount; i++) {
+            page = downloadPage(String.format(url, pageCount));
+            vacancyList.addAll(fillVacancy2(getLinksFromPage(page)));
+        }
+
+        return vacancyList;
+    }
+
+    private int getPageCount(HtmlPage page) {
+        return Integer.parseInt(page.getByXPath(JOBS_PAGINATION_LIST_XPATH).get(0).toString().replaceAll("\\D+", "")) / 100;
+    }
+
+    private DomNodeList<DomNode> getLinksFromPage(HtmlPage page) {
+        return page.querySelectorAll(".JobsListings__link");
+    }
+
+    private List<Vacancy> fillVacancy2(DomNodeList<DomNode> links) {
+        List<Vacancy> vacancies = new ArrayList<>();
+        links.forEach(link ->
+                vacancies.add(fillVacancyDTO(link.getAttributes().getNamedItem("href").getNodeValue().substring(13)))
+        );
+
+        return vacancies;
+    }
+
+    @SneakyThrows
+    private Vacancy fillVacancyDTO(String link) {
+        VacancyDTO vacancyDTO = new VacancyDTO();
+
+        HtmlPage page = downloadPage(VACANCY_URL + link);
+        DomNodeList<DomNode> description = page.querySelectorAll(".ArticleMarkdown");
+        DomNodeList<DomNode> jobElements = page.querySelectorAll(".JobDetailCardProperty__title");
+        HtmlHeading1 title = (HtmlHeading1) page.getByXPath(VACANCY_TITLE_XPATH).get(0);
+
+        vacancyDTO.setDescription("");
+
+        description.get(0).getChildren().forEach(
+                iter -> vacancyDTO.setDescription(vacancyDTO.getDescription() + iter.asNormalizedText())
+        );
+
+        vacancyDTO.setLink(link);
+        vacancyDTO.setOfficeLocation(List.of(jobElements.get(0).getNextElementSibling().getTextContent().replace(" or ", "").split(", ")));
+        vacancyDTO.setJobType(jobElements.get(jobElements.size() - 1).getNextElementSibling().getTextContent());
+        vacancyDTO.setTeam(jobElements.get(jobElements.size() - 2).getNextElementSibling().getTextContent());
+        vacancyDTO.setRoles(title.getTextContent());
+
+        if (jobElements.size() == 4) {
+            vacancyDTO.setHasRemote(true);
+            vacancyDTO.setRemoteLocation(List.of(jobElements.get(1).getNextElementSibling().getTextContent().replace(" or ", "").split(", ")));
+        }
+
+        System.out.println(vacancyDTO);
+
+        return fillVacancy(vacancyDTO);
+    }
+
+    private Vacancy fillVacancy(VacancyDTO vacancyDTO) {
+        Vacancy vacancy = new Vacancy();
+
+        List<String> locations = new ArrayList<>();
+        String description = vacancyDTO.getDescription() +
+                (vacancyDTO.getHasRemote() ? "Has Remote: true" : "Has Remote: false");
```

Comment:
If you write something like this, it damages readability a lot. Please, move all such things to their own methods with the descriptive names

Besides, we don't need this info in the description. I guess, there's a task on the board to introduce isRemote boolean field in DB for a vacancy